src/tooltip/tooltip.spec.ts

```diff
@@ -60,6 +61,78 @@ describe('ngb-tooltip-window', () => {
     expect(fixture.nativeElement).toHaveCssClass('my-custom-class');
   });
 
+  it('should allow to change custom class to new one', () => {
+    const fixture = TestBed.createComponent(NgbTooltipWindow);
+    fixture.componentInstance.tooltipClass = 'first-custom-class';
+    fixture.detectChanges();
+    fixture.componentInstance.tooltipClass = 'second-custom-class';
+    fixture.detectChanges();
+    fixture.componentInstance.tooltipClass = 'third-custom-class';
+    fixture.detectChanges();
+
+    expect(fixture.nativeElement).toHaveCssClass('third-custom-class');
+  });
+
+  describe('ngb-tooltip-window tooltipClass change detection', () => {
```

Comment:
Check my other comment, it's Angular who will call change detection if any of the inputs change regardless of what you do. So this `describe` looks useless to me.

Reply:
redacted, thanks!